java/src/jmri/jmrit/display/layoutEditor/LayoutTrackEditors.java

```diff
@@ -611,9 +611,6 @@ protected void editLayoutTurnout(@Nonnull LayoutTurnout layoutTurnout) {
         editLayoutTurnout1stTurnoutComboBox.addPopupMenuListener(
                 layoutEditor.newTurnoutComboBoxPopupMenuListener(editLayoutTurnout1stTurnoutComboBox, currentTurnouts));
 
-        editLayoutTurnout2ndTurnoutComboBox.addPopupMenuListener(
```

Comment:
@dsand47 Is it correct to be removing this listener?

Reply:
Yes.

The listener is used to filter the turnouts which already exist on a LE panel.

In order to link two turnouts all of the turnouts have to be included in the 2nd combo so that the related turnout can be selected.   The relationship is maintained within the parent LayoutTurnout.

This does not affect the block routing requirement for no duplicate turnouts on a LE panel.